src/Helpers/Endpoint.php

```diff
@@ -36,11 +36,7 @@ public function __construct(Shopify $client)
      */
     public function __get($property)
     {
-        if (isset($this->$property)) {
-            return $this->$property;
-        }
-
-        return $this->client->__get($property);
+        return $this->$property ?? $this->client->__get($property);
```

Comment:
What is the property is null?

<img width="214" alt="Screen Shot 2020-04-13 at 9 15 01 AM" src="https://user-images.githubusercontent.com/470255/79123075-4622a300-7d67-11ea-875c-25231cf7f863.png">

Reply:
Looks like `isset` also returns false on a variable that's set to null.
<img width="195" alt="Screen Shot 2020-04-13 at 5 41 27 PM" src="https://user-images.githubusercontent.com/49741676/79163945-0337ee00-7dae-11ea-93ef-32e6bf4c662b.png">